pac/managing-pipeline-runs-pac.adoc

```diff
@@ -0,0 +1,42 @@
+:_content-type: ASSEMBLY
+[id="managing-pipeline-runs-pac_{context}"]
+include::_attributes/common-attributes.adoc[]
+= Managing pipeline runs
+:context: managing-pipeline-runs-pac
+
+toc::[]
+
+[role="_abstract"]
+Using {pac}, you can create pipelines in your code repository and run these pipelines.
+
+include::modules/op-creating-pipeline-run-using-pipelines-as-code.adoc[leveloffset=+1]
+
+.Additional resources
+
+* link:https://github.com/google/cel-spec/blob/master/doc/langdef.md[CEL language specification]
```

Comment:
@mramendi Refer to the admonition related to github.com: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites

Global comment for all similar instances.

Reply:
Not in scope as the PR is restructure-only and the link was in the previous content (it shows as new simply because of a split assembly). Replacing this link would require cycling back to SME/QE.